mindsdb/integrations/handlers/dspy_handler/requirements.txt

```diff
@@ -0,0 +1,11 @@
+openai==1.6.1
+wikipedia==1.4.0
+tiktoken==0.5.2
+anthropic>=0.26.1
+litellm==1.35.0
+chromadb # Knowledge bases.
+langchain-experimental==0.0.58
+dspy-ai
```

Comment:
As with other requirements, we probably want to pin `dspy` and `dspy-ai` to guarantee an update there won't suddenly break our integration.